### PR TITLE
dialog: init at 1.2-20150225

### PR DESCRIPTION
**Pull request by spacefrogg**
Create ncurses dialogs from shell scripts.

### CODE REVIEW

File: pkgs/development/tools/misc/dialog/default.nix [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    --with-ncurses${optStr unicodeSupport "w"}
+  '';
+
+  installTargets = "install${optStr withLibrary "-full"}";
+
+  meta = {
+    homepage = http://invisible-island.net/dialog/dialog.html;
+    description = "Display dialog boxes from shell";
+    license = stdenv.lib.licenses.lgpl21Plus;
+
```
Comment (jagajaga) [inline pkgs/development/tools/misc/dialog/default.nix]: Remove this empty line.

**Comment by jagajaga** [comment]
@garbas why did you merge before this typo was fixed?

**Comment by garbas** [comment]
@jagajaga please check the commit. i've fixed it there.

i dont think it is nice to ask for every little change. especially if it takes 2 seconds to fix it yourself. that is after commenting what can be approved. this way pull requests are also used as a teaching tool and are not a bottleneck of accepting new commit.

**Comment by jagajaga** [comment]
Fixing it by yourself is not good because
* people will send us not properly formed commits and we will have to fix them everytime. 
* if you fix and push, the author of the commit will be the same, but that can't be seen in the profile of that person. Some people don't like that, @matthiasbeyer for example.

**Comment by garbas** [comment]
@jagajaga i don't think you've really looked at the commit

* author of the commit is the same (i'm just the commiter) and github shows this respectively. just check on @spacefrogg timeline (https://github.com/NixOS/nixpkgs/commits?author=spacefrogg)
* people learn, give them a chance.

**Comment by jagajaga** [comment]
PR is not marked as "merged", that's what I wanted to say.

People learn by their mistakes, and the best way is to solve these 
mistakes, it's not good to just look at final result :)

**Pull request closed without merging**